### ISSUE TITLE
BUG: Nothing can be edited inside the link element

### ISSUE THREAD

**Issue by Ameeko** [artf/grapesjs#4154]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome v98

### Reproducible demo link

https://grapesjs.com/demo.html

### Describe the bug

When I insert a **text** element into a **link block** element, 
the text element is no longer editable. 

Double-clicking on text element does not switch to edit mode, but selects the (container) link block element.

![image](https://user-images.githubusercontent.com/61547701/155326378-5404b8d4-6c92-4708-9e49-d439aa6b31a8.png)
![image](https://user-images.githubusercontent.com/61547701/155326454-5ae1dfdb-7b82-46a7-8ccc-17149110237e.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lofcz** [comment]
duplicate of: https://github.com/artf/grapesjs/issues/4145
tl;dr: build grapejs locally as fix is currently unreleased or downgrade to 0.18.2

**Comment by artf** [comment]
Closing as duplicate of #4145
